hack/setup.py

```diff
@@ -0,0 +1,65 @@
+#
+# File originaly from https://github.com/konveyor/tackle-test-generator-cli/blob/main/setup.py
+#
+# TODO: Required dependencies list should be minimized more if possible (L30-48).
+#
+
+import tkltest
+from setuptools import setup, find_packages
+from setuptools.command.install import install
+
+
+class TkltestInstall(install):
+    """Install and download nltk data"""
+    def run(self):
+        install.run(self)
+#        import nltk
+#        nltk.download('stopwords')
+#        nltk.download('averaged_perceptron_tagger')
+#        nltk.download('wordnet')
+#        nltk.download('words')
+#        nltk.download('omw-1.4')
+
+
+setup(
+    name='tkltest',
+    version=tkltest.__version__,
+    description='Command-line interface for generating and executing test cases on'
+                'two application versions and performing differential testing',
+    packages=find_packages(),
+    install_requires=[
```

Comment:
Required dependencies list should be minimized more to cover only unit test dependencies if possible (L30-48), this is an upcoming work.